docs/Epita/dotnet.mdx

```diff
@@ -8,112 +8,100 @@ tags:
 - .NET
 - C#
 - Jetbrains
-description: C#/.NET sur MacOS avec Jetbrains Rider
+description: C#/.NET on MacOS with Jetbrains Rider
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# C#/.NET sur MacOS
+# C#/.NET on MacOS
 
-## Prérequis
+## Prerequisites
 
 - [Homebrew](../../Tools/Homebrew)
-- ou [Nix](../../Tools/Nix)
-- Rider (ou Visual Studio, Rider est recommandé par l'école, car compatible avec Linux, et est accessible pendant les examens)
+- or [Nix](../../Tools/Nix)
+- Rider (or Visual Studio, Rider is recommended by the school, because it is compatible with Linux, and is accessible during exams)
 
 ## Introduction
 
-.NET est un framework développé par Microsoft, qui permet de développer des applications en C#.
-.NET est disponible sur MacOS, Linux et Windows.
-C'est le framework utilisé par l'école pour les cours de C# de première année à l'EPITA, encadrés par les assistants (ACDC).
+.NET is a framework developed by Microsoft, which allows the development of applications in C#. .NET is available on MacOS, Linux, and Windows.
+It is the framework used by the school for the first year C# courses at EPITA, supervised by the assistants (ACDC).
+
 
 ## Installation
 
-Pour installer .NET, plusieurs moyens s'offrent à vous.
-Vous pouvez installer Rider, qui vous proposera d'installer .NET au premier lancement, ou bien l'installer
-manuellement en ligne de commande. Commençons par installer Rider.
+To install .NET, you have several options.
+You can install Rider, which will propose to install .NET during the first launch, or install it manually using the command line.
+Let's start by installing Rider.
 
 ### Rider
-
-Rider est un IDE développé par Jetbrains, qui est compatible avec .NET, C# et Mono.
-Il est disponible sur MacOS, Linux et Windows.
-
-Pour l'installer, il suffit de se rendre sur le site de [Jetbrains](https://www.jetbrains.com/rider/) et de télécharger la version pour MacOS.
-
+Rider is an IDE developed by JetBrains, which is compatible with .NET, C#, and Mono. It is available on MacOS, Linux, and Windows.
+To install it, simply go to the JetBrains website and download the macOS version.
```

Comment:
```suggestion
To install it, simply go to the [JetBrains](https://www.jetbrains.com/rider/) website and download the macOS version.
```

Je ne sais pas si je ne sais pas lire, mais il me semble qu'il manque les liens

Reply:
Reolved in [24a239d](https://github.com/EpiMac/docs/pull/15/commits/24a239d6f807caf054a562a05eb88cf800e2e7a3)